### ISSUE TITLE
[WCAG] Ensure keyboard navigation - Layers Panel

### ISSUE THREAD

**Issue by kenchase** [Canadian-Geospatial-Platform/geoview#3219]
### Is there an existing issue for this?

- [x] I have searched the existing issues

### Current Behavior

Footer: LayerList panel items can't be pressed using the spacebar.

### Expected Behavior

Footer: LayerList panel items should be pressable using the spacebar.
